extensions/ql-vscode/package.json

```diff
@@ -461,13 +461,17 @@
         "command": "codeQLQueryHistory.showQueryText",
         "title": "Show Query Text"
       },
+      {
+        "command": "codeQLQueryHistory.exportCsvResults",
+        "title": "Export Results (CSV)"
+      },
       {
         "command": "codeQLQueryHistory.viewCsvResults",
```

Comment:
Minor: I see that the command itself is still called `viewCsvResults`. Is it worth changing that to `Alerts` too?
(possibly not, since it isn't user-visible 😄)

Reply:
Good point, probably worth changing it too even it isn't user-facing to avoid confusion.